release_notes.md

```diff
@@ -2,4 +2,5 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR/#issue)
 -->
-- Storage extensions moved to 5.0.0
\ No newline at end of file
+- Storage extensions moved to 5.0.0
+- SignalR extensions added some convenient POJOs (PR #774)
```

Comment:
We should also consider adding sample usage of this types in the extension samples.

Reply:
Currently the extension samples use package reference, so we cannot add new usages to the samples until they are released.